Plan/fabric/build.gradle

```diff
@@ -77,8 +78,6 @@ shadowJar {
     exclude "jakarta/xml/**/*"
     exclude "javassist/**/*"
 
-    relocate 'org.slf4j', 'plan.org.slf4j'
-
```

Comment:
SLF4J needs to be relocated, because unrelocated dependencies might cause clashes with other plugins using the same dependency.

This unfortunately causes other issues, mainly SLF4J not finding the relocated logging provider due to some problems with class loading, which causes the aforementioned issue #2517.